internal/apps/dop/providers/issue/core/query/get.go

```diff
@@ -41,3 +41,27 @@ func (p *provider) GetIssue(id int64, identityInfo *commonpb.IdentityInfo) (*pb.
 	}
 	return issue, nil
 }
+
+func (p *provider) BatchGetIssue(ids []int64, identityInfo *commonpb.IdentityInfo) ([]*pb.Issue, error) {
+	// 请求校验
+	if len(ids) == 0 {
+		return nil, apierrors.ErrBatchGetIssue.MissingParameter("ids")
+	}
+	// 查询事件
+	model, err := p.db.BatchGetIssue(ids)
+	if err != nil {
+		if gorm.IsRecordNotFoundError(err) {
+			return nil, apierrors.ErrBatchGetIssue.NotFound()
+		}
+		return nil, apierrors.ErrBatchGetIssue.InternalError(err)
+	}
+	issues := make([]*pb.Issue, 0)
+	for _, v := range model {
+		issue, err := p.Convert(v, identityInfo)
+		issues = append(issues, issue)
+		if err != nil {
+			return nil, apierrors.ErrBatchGetIssue.InternalError(err)
+		}
+	}
+	return issues, nil
+}
```

Comment:
is `BatchGetIssue` needed? may be you could use `Paging` to batch get issues

Reply:
I think it is necessary because the paging conditions are not suitable for obtaining specific batch items